packages/gatsby-theme-docs/gatsby-browser.js

```diff
@@ -13,9 +13,9 @@ import { SWRConfig } from 'swr';
 import { docsCache } from './utils/create-emotion-cache';
 import { PortalsContainer } from '@commercetools-docs/ui-kit';
 import * as Sentry from '@sentry/browser';
-import '@fontsource/roboto/latin-400.css';
-import '@fontsource/roboto/latin-500.css';
-import '@fontsource/roboto/latin-700.css';
+import '@fontsource/inter/latin-400.css';
+import '@fontsource/inter/latin-500.css';
+import '@fontsource/inter/latin-700.css';
```

Comment:
maybe try  `latin-ext-400.css` for the disambiguation issue.   (also in the 500 and 700 weights if needed there)

Reply:
Didn't work unfortunately.